modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS). You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+.Procedure
+
+. Create a `Secret` YAML file that uses the `aws_access_key_id` and `aws_secret_access_key` fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+----
+
+
+. Create the secret. For example:
++
+[source,terminal]
+----
+$ oc apply -f cw-secret.yaml
```

Comment:
Commands with `oc` means that you have to be on an OpenShift cluster. Would your users know this and already be on an OpenShift cluster if they were to start with this documentation? 

At least for me, coming into this doc cold and reading the intro statement, I didn't think I had to be on an OpenShift cluster in order to run the commands in the procedure.

Reply:
Consider saying, `Create the secret YAML file` In Step 1, you referred to it as `Secret YAML file` so it might be clearer to the user if you refer to it the same way in Step 2. 

Also, would it need to be `**S**ecret` like how it was written out in Step 1? I'm not sure why it's not capitalized in Step 2.

Reply:
@jc-berger Given these are OpenShift docs, specifically about collection of logs from an OpenShift cluster, I think it's reasonable to assume that people would have an OpenShift cluster before they concern themselves with collecting logs from it.

Reply:
Yes. Logging users understand that this is all being done in an OpenShift cluster.